### PR TITLE
Test single letter increment of tag to fix build.

### PR DESCRIPTION
**Pull request by the-snowwhite**

### CODE REVIEW

File: bundles/org.eclipse.swt/buildSWT.xml [hunk 1]
```diff
@@ -291,19 +291,27 @@
...
+	            var found = false;
+	            var suffix = 'a';
+	            while (!found) {
+	                var t = swt_version + suffix;
+	                if (!tags.match(t)) {
+	                    project.setProperty("swt_new_tag", t);
+	                    found = true;
+	                } else {
+	                    if (suffix === 'z') {
+	                        suffix = 'a'; // Reset to 'a' after 'z'
```
Comment (mickaelistria) [inline bundles/org.eclipse.swt/buildSWT.xml:308]: In existing code, if we've reached the case of 'z', it is most likely that the case of 'a' (`i=97`) and all other letters already reported that the suffix is already used.
You would basically want to write an algorithm that iterates through the sequences a..z,aa...az, ba...bz, ..., za...zz, aaa...aaz, aba...abz, ..., aza,...,azz, baa...baz and so on. It's a lot of smartness to put in a build scripts, particularly if we don't really need the tags.
Maybe just a timestamp suffix would be enough?

Comment (the-snowwhite) [reply bundles/org.eclipse.swt/buildSWT.xml:308]: Does github support numbers in suffixes ?

Comment (mickaelistria) [reply bundles/org.eclipse.swt/buildSWT.xml:308]: Git and GitHub don't have a concept of suffix, it's just some characters in the tag name; and yes, they do support numbers as characters in tag names.

Comment (the-snowwhite) [reply bundles/org.eclipse.swt/buildSWT.xml:308]: We would'nt have this problem if the v4963r3 part of the tags were sure to be updated before 26 commits,
What mechanism updates this part or the tag ?

**Comment by the-snowwhite** [comment]
This build seems stuck.

**Pull request closed without merging**